Plugins/MatrixPlugin/MatrixForm.cs

```diff
@@ -0,0 +1,257 @@
+using Chummer;
+using Chummer.Backend.Equipment;
+using System;
+using System.Collections.Generic;
+using System.Windows.Forms;
+
+namespace MatrixPlugin
+{
+    public partial class MatrixForm : Form
+    {
+        private readonly MatrixLogic logic;
+
+        public MatrixForm(MatrixLogic logic)
+        {
+            this.logic = logic;
+            
+            InitializeComponent();
+            InitializeContent();
+            InitializeBinding();
+
+            void InitializeContent()
+            {
+                foreach(var person in logic.Persons)
+                    if (person is Gear gear)
+                        listCyberDecks.Items.Add(gear.Name);
+                foreach (var software in logic.Software)
+                    listSoftware.Items.Add(software.Name);
+                foreach (var action in logic.Actions)
+                    cbActions.Items.Add(action.Name);
```

Comment:
This does not seem like it would work well with people adding or removing gear from their characters, as well potentially adding or removing `IHasMatrixAttribute` objects of other kinds, like say a deltaware cyberarm (DR5!) with a personafix electronic modification.

Reply:
Add binding on change gear of character in (https://github.com/chummer5a/chummer5a/pull/4563/commits/cd67c543cb0c14c2a3d4068113bf15cca275a6f9)